home/views.py

```diff
@@ -79,24 +79,24 @@ def dashboard(request):
         context['cw_hw'] = cw_hw
 
         # Students Attendance
-        student_attendence = StudentAttendence.objects.filter(cal_date=calendar, present=True).order_by('student__school_class')
-        context['student_attendence'] = student_attendence
+        student_attendance = StudentAttendance.objects.filter(cal_date=calendar, present=True).order_by('student__school_class')
```

Comment:
Changing variable name is fine but don't change `StudentAttendence` as it is an import.